src/pushNotification.ts

```diff
@@ -0,0 +1,39 @@
+import walletApi from './wallet/api/walletApi';
+import { PushNotificationResult, PushRegisterRequestData, PushUnregisterRequestData, PushUpdateRequestData } from './wallet/types';
+import HathorWalletServiceWallet from './wallet/wallet';
+
+export class PushNotification {
+
+  /**
+   * Register the device to receive push notifications
+   *
+   * @memberof HathorWalletServiceWallet
```

Comment:
```suggestion
   * @memberof PushNotification
```

We must update this in all the methods